File: articles/algorithms-in-labor-management-and-organizational-systems/fortran/workload_burden.f90
program workload_burden_example
  implicit none
  real :: pace, hours, fatigue, schedule_volatility, burden
  pace = 0.84
  hours = 0.72
  fatigue = 0.70
  schedule_volatility = 0.78
  burden = (pace + hours + fatigue + schedule_volatility) / 4.0
  print *, "workload_burden_score=", burden
end program workload_burden_example

File: articles/algorithms-in-labor-management-and-organizational-systems/rust/workload_burden.rs
fn main() {
    let pace = 0.84_f64;
    let hours = 0.72_f64;
    let fatigue = 0.70_f64;
    let schedule_volatility = 0.78_f64;
    let burden = (pace + hours + fatigue + schedule_volatility) / 4.0;
    println!("workload_burden_score={:.4}", burden);
}
